### PR TITLE
fix innerHTML and outerHTML methods and add tests

### PR DESCRIPTION
**Pull request by mindywhitsitt**
Breaking change in Protractor 5.0.0. The Protractor syntax is a lot uglier now.

@scriby would you please take a look at this as well?

### CODE REVIEW

**Review by casteaux — DISMISSED** [review]
I love these tests

File: test/spec/protractor_sync_test.ts [hunk 1]
```diff
@@ -193,6 +193,29 @@ describe('Protractor extensions', () => {
...
+
+    it('outerHTML', createTest(() => {
+      //Make sure we are starting on a fresh page
+      browserSync.get('data:,');
+
+      appendTestArea({
+        innerHtml: '<div class="div-with-text">Some Text In The Div</div>'
+      });
+
+      expect(elementSync.findVisible('.div-with-text').getOuterHtml()).toBe('<div class="div-with-text">Some Text In The Div</div>');
```
Comment (shermaneric) [inline test/spec/protractor_sync_test.ts:217]: Nice find Mindy! 
Minor:  Do we want to have the tests use this ```Some <b> HTML </b> In The Div```  to catch any inner HTML tags?  I'm fine if not.   
All looks good.  Approving.

**Review by shermaneric — DISMISSED** [review]

**Comment by scriby** [comment]
Did you look into why the base protractor methods don't work anymore? I'm going to take a peek myself, seems weird.

**Comment by scriby** [comment]
Oh weird, they removed getInnerHtml and getOuterHtml from selenium/protractor. I wonder what the reasoning was...

**Comment by scriby** [comment]
🚢

**Comment by mindywhitsitt** [comment]
Yeah, not sure why they did that. I didn't dig any further than the changelog that showed what the new syntax was.

File: test/spec/protractor_sync_test.ts (hunk 1 above)
Comment (mindywhitsitt) [reply test/spec/protractor_sync_test.ts:217]: Sure. That would probably demo what it's doing a little more explicitly.

**Review by casteaux — APPROVED** [review]

**Review by shermaneric — APPROVED** [review]

**Pull request merged**